pkg/recipes/terraform/execute.go

```diff
@@ -218,6 +224,19 @@ func (e executor) setEnvironmentVariables(ctx context.Context, tf *tfexec.Terraf
 	return nil
 }
 
+// splitEnvVar splits a slice of environment variables into a map of keys and values.
+func splitEnvVar(env []string) map[string]string {
```

Comment:
We can create unit tests specific to this function.

Reply:
updated